Composer/packages/adaptive-flow/src/adaptive-flow-editor/renderers/NodeWrapper.tsx

```diff
@@ -38,6 +38,8 @@ export const ActionNodeWrapper: FC<NodeWrapperProps> = ({ id, tab, data, onEvent
   const nodeFocused = focusedId === id || focusedEvent === id;
   const nodeDoubleSelected = tab && nodeFocused && tab === focusedTab;
   const nodeSelected = selectedIds.includes(id);
+  const nodeId = `action-${selectableId}`;
+  const actionNode = document.getElementById(nodeId);
```

Comment:
Where is the `nodeId` built and can we use `actionRef` to focus the dom element?

Reply:
`nodeId` set as element id in `<div id={nodeId} ... />`
I see that `actionRef` is a callback and can not used at element. 
```
  const {
    shellApi: { addCoachMarkRef },
  } = useShellApi();
  const actionRef = useCallback((action) => {
    addCoachMarkRef({ action });
  }, []);
```